projects/go-lib/src/lib/components/go-header-bar/go-header-bar.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../styles/_variables.scss";
+.go-header-bar {
+  margin: -2rem;
+  height: 68px;
+  background: white;
+  box-shadow: 0 0 2px rgba(0, 0, 0, 0.12), 0 2px 2px rgba(0, 0, 0, 0.24);
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  position: fixed;
+}
+
+.go-page-header__title {
+  padding: 0 2rem;
+  margin-bottom: -0.25rem;
+}
+
+.go-page-header__back-arrow {
+  cursor: pointer;
+  align-items: center;
+  padding: 1.5rem 1rem;
+  margin-right: -1rem;
+  border-right: solid 1px lighten($base-light-secondary, 17%);
+  flex-basis: 40px;
```

Comment:
Could we utilize one of the `$column-gutter` variables for this? Or do we really need this item?